src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1048,12 +1048,35 @@ public override void VisitParenthesizedExpression(IParenthesizedExpression opera
             base.VisitParenthesizedExpression(operation);
         }
 
-        public override void VisitLateBoundMemberReferenceExpression(ILateBoundMemberReferenceExpression operation)
+        public override void VisitDynamicMemberReferenceExpression(IDynamicMemberReferenceExpression operation)
         {
-            LogString(nameof(ILateBoundMemberReferenceExpression));
-            LogString($" (Member name: {operation.MemberName})");
+            LogString(nameof(IDynamicMemberReferenceExpression));
+            LogString($" (Member name: {operation.MemberName}, Containing Type: {(object)operation.ContainingType ?? "null"})");
             LogCommonPropertiesAndNewLine(operation);
 
+            if (!operation.TypeArguments.IsDefaultOrEmpty)
```

Comment:
You can probably use the helper I am adding here: https://github.com/dotnet/roslyn/pull/20962/files#diff-7112e58be0d461a84d9f9b42fd765780R290 to ensure the output is consistent with any array dumped by the dumper.